frontend/src/components/ChatMessage.tsx

```
@@ -1,18 +1,102 @@
+    case "edit_file":
+      if (result.error) {
+        return <div className="error-message">Error: {result.error}</div>;
+      }
+      return (
+        <div className="tool-content">
+          <p>{result.message}</p>
+        </div>
+      );
+
+    case "run_bash_command":
+      const { stdout, stderr } = result;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)